docs/book/src/reference/scopes.md

```diff
@@ -0,0 +1,118 @@
+namespace-scoped or cluster-scoped, affecting their availability
+across the cluster.
+
+## Configuring Manager Scope
+
+Managers can operate under different scopes depending on
+the resources they need to handle:
+
+### (Default) Watching All Namespaces
+
+By default, if no namespace is specified, the manager will observe all namespaces.
+This is configured as follows:
+
+```go
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+...
+})
+```
+
+### Watching a Single Namespace
+
+To constrain the manager to monitor resources within a specific namespace, set the Namespace option:
+
+```go
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+...
+   Cache: cache.Options{
+      DefaultNamespaces: map[string]cache.Config{"operator-namespace": cache.Config{}},
+   },
+})
+```
+
+### Watching Multiple Namespaces
+
+A manager can also be configured to watch a specified set of namespaces using [Cache Config][CacheConfig]:
+
+```go
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+...
+Cache: cache.Options{
+    DefaultNamespaces: map[string]cache.Config{
+        "operator-namespace1": cache.Config{},
+        "operator-namespace2": cache.Config{},
+        },
+    },
+})
+```
+
+## Configuring CRD Scope
+
+The scope of CRDs determines their visibility either within specific namespaces or across the entire cluster.
+
+### Namespace-scoped CRDs
+
+Namespace-scoped CRDs are suitable when resources need to be isolated to specific namespaces.
+This setting helps manage resources related to particular teams or applications.
+However, it is important to note that due to the unique definition of CRDs (Custom Resource Definitions) in Kubernetes, testing a new version of a CRD is not straightforward. Proper versioning and conversion strategies need to be implemented (example in our [kubebuilder tutorial][kubebuilder-multiversion-tutorial]), and coordination is required to manage which manager instance handles the conversion (see the official [kubernetes documentation][k8s-crd-conversion] about this).
+Additionally, the namespace scope must be taken into account for mutating and validating webhook configurations to ensure they are correctly applied within the intended scope. This facilitates a more controlled and phased rollout strategy.
+
+### Cluster-scoped CRDs
+
+For resources that need to be accessible and manageable across the entire cluster,
+such as shared configurations or global resources, cluster-scoped CRDs are used.
+
+#### Configuring CRDs Scopes
+
+**When the API is created**
+
+The scope of a CRD is defined when generating its manifest.
+Kubebuilder facilitates this through its API creation command.
+
+By default, APIs are created with CRD scope as namespaced. However,
+for cluster-wide you use `--namespaced=false`, i.e.:
+
+```shell
+kubebuilder create api --group cache --version v1alpha1 --kind Memcached --resource=true --controller=true --namespaced=false
```

Comment:
The --namespaced=false parameter is optional. The default value is false. You can set it to true only when you are trying it out.